app/src/main/java/com/melvin/ongandroid/model/entities/activities/Activitie.kt

```diff
@@ -0,0 +1,14 @@
+package com.melvin.ongandroid.model.entities.activities
+
+import android.text.Html
+
+data class Activitie(
+    val id: Int,
+    val name: String,
+    val slug: String,
+    val description: String,
+    val image: String,
+){
+    val rawDescription: String
+        get() = Html.fromHtml(this.description, Html.FROM_HTML_MODE_COMPACT).toString()
```

Comment:
Mirar esto: https://stackoverflow.com/questions/37904739/html-fromhtml-deprecated-in-android-n